content/800-data-platform/200-projects/100-add-project/110-create-new-project-from-template.mdx

```diff
@@ -0,0 +1,107 @@
+
+        • The GitHub repository name is pre-filled based on the display name, where the repository name is lowercase-only and spaces are replaced with hyphens.<br /><br />
+        • You cannot create a project with a repository name duplicating that of an existing repository in the select account or organization.<br />
+        </Admonition>
+
+    5. Click **Next**.
+        <img
+        src="https://res.cloudinary.com/prismaio/image/upload/v1666700351/docs/pdp/create-manage-projects/create-project-01-configure-project_s4s19e.png"
+        alt="Prisma Data Platform - Create new project - Name your project and create a new GitHub repository"
+        width="750px"
+        />
+3. On the _Select template_ page, select an application template for your first project.
+    1. Select one of the available application templates. When you select a template, you can see a preview of its Prisma schema file below.
+        * **Blog**. Schema for a blog built with Next.js.
+        * **Music Streaming Service**. Schema for a fictional music streaming service with a REST API.
+        * **Rentals Platform**. Schema for a fictional hotel with a REST API.
+        * **SaaS**. Schema for an identity service with a SaaS with a REST API.
+        * **URL Shortener**. Schema for a URL shortener with a REST API.
+        * **Empty**. An empty schema with no models, to build your own schema.
+    2. (Optional) Clear **Include sample data** if you do not want to seed the database with sample data.
+
+        <img
+            src="https://res.cloudinary.com/prismaio/image/upload/v1666649988/docs/pdp/create-manage-projects/create-project-02-clear-include-sample-data_qvdps1.png"
+            alt="Prisma Data Platform - Create a new project - Select a template project and seed your database"
+            width="750px"
+        />
+    3. Click **Next**.
+4. On the _Configure environment_ page, connect your database and configure the Data Proxy location.
+    1. In **Connection string**, paste your database connection string.
+        <Admonition type="alert">
+
+        **Important**<br /><br />
+        If you provide a connection string to a non-empty database (with existing tables and data), the project creation fails.<br /><br />
+
+        To add a project with an existing database, see [Import an existing project with an existing database](/data-platform/projects/add-project/import-existing-project-existing-db).
+        </Admonition>
+    2. Under _Data Proxy_, from the **Location** drop-down menu, select the geographic location for the Data Proxy.
+        <Admonition>
+        
+        **Note**<br /><br />
+        Select a Data Proxy location that is close to the geographic location of your database.<br /><br />
+        If you select a Data Proxy location that is remote from your database, then when your application sends requests to your database through the Data Proxy, the responses might be slow or time out due to higher latency.
```

Comment:
"Might be slow or time out due to higher latency" - could we change to something more lighter? Slow sounds really slow. Maybe: "there may be a delay...". This is a nitty picky thing, just to consider :)